doc/source/ray-air/check-ingest.rst

```diff
@@ -97,7 +97,8 @@ Bulk Ingest
 ~~~~~~~~~~~
 
 By default, AIR loads all Dataset blocks into the object store at the start of training. This provides the best performance if the
-cluster has enough aggregate memory to fit all the data blocks in object store memory. Note that data often requires more space
+cluster has enough aggregate memory to fit all the data blocks in object store memory, or if your preprocessing step is expensive
+and you don't want it to be re-run on each epoch. Note that data often requires more space
```

Comment:
If it's bulk ingest, there is no concept of "epoch" since it's DatasetPipeline related?

Reply:
In ML terms, an epoch is a pass over the whole dataset, so it's still there from the user's view.